src/tests/ftest/directory_tree.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
```

Comment:
Python scripts cannot be packed with this.  You will need to add this script to the list at https://github.com/daos-stack/daos/blob/e84c9636f2b445bb9758ee9575abfc16c0f25372/utils/rpms/daos.spec#L365.

We really should make that a generic search and replace in the whole ftest tree.

Reply:
This will be fine, the RPM will change it at some point before install.  The trick is to run it as `python directory_tree.py` for it to work properly.

Reply:
It's probably simpler to not make the file executable, remove the shebang, and execute with `python3 directory_tree.py`

Reply:
And for robustness in remote execution, we can use `sys.executable` to get the path to the current running python
```
>>> import sys
>>> sys.executable
'/usr/bin/python3'
```
Which works if running in a virtual env too

Reply:
Thanks, I had forgot about that.

Reply:
I've moved these changes to https://github.com/daos-stack/daos/pull/14262 where the src/tests/ftest/directory_tree.py file no longer uses `#!/usr/bin/env python3` and instead we execute the command using the python executable as suggested by Dalton.